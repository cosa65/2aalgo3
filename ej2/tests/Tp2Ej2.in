10 10
0 10 9 1
4 4
0 1 1 2; 1 2 3 1; 2 3 3 4; 3 3 2 1
5 5
0 2 2 3; 2 4 3 2; 3 1 4 4; 4 3 4 2
10 10
0 0 1 0; 1 0 2 0; 2 0 3 0; 3 0 4 0; 4 0 5 0; 5 0 6 0; 6 0 7 0; 7 0 8 0; 8 0 9 0
10 10
0 1 1 1; 1 1 2 1; 2 1 3 1; 3 1 4 1; 4 1 5 1; 5 1 6 1; 6 1 7 1; 7 1 8 1; 8 1 9 1
10 10000
0 0 1 0; 1 0 2 0; 2 0 3 0; 3 0 4 0; 4 0 5 10000; 5 10000 6 10000; 6 10000 7 5000; 7 5000 8 5000; 8 5000 1 10000; 1 10000 9 10000