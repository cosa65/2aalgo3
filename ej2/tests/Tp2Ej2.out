21
8
11
28
28
20
